src/components/tx/SpendingLimitRow/index.tsx

```diff
@@ -28,25 +28,28 @@ const SpendingLimitRow = ({
           name={SendAssetsField.type}
           render={({ field: { onChange, ...field } }) => (
             <RadioGroup
-              onChange={async (e) => {
+              onChange={(e) => {
+                onChange(e)
+
```

Comment:
Did `Promise.resolve()` not work?

Reply:
I replaced it with a timeout because it's more explicit/idiomatic. Waiting for a promise that does nothing is not self-explanatory.